locustempus/templates/main/course_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<h1>Course Delete Form</h1>
+<p>Create all the courses.</p>
+<form action="." method="post">
+    {% csrf_token %}
+    <p>Are you sure you want to delete "{{ course.title }}"</p>
+    <input type="submit" value="Confirm">
```

Comment:
You'll want a Cancel on this template that navigates back to the course detail or course list page.